pkgs/by-name/ed/edukai/package.nix

```
@@ -20,12 +20,9 @@ stdenvNoCC.mkDerivation rec {
   '';
 
   meta = {
-    description = "The MOE Standard Kai Font, a Chinese font by the Ministry of Education, ROC (Taiwan)";
+    description = "The MOE Kai font, a regular-script-style Chinese character typeface";
```

Current version of [`pkg/README.md`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes) forbids `meta.description` starting with an article.

```suggestion
    description = "MOE Kai font, a regular-script-style Chinese character typeface";
```